map2loop/thickness_calculator.py

```diff
@@ -1,9 +1,17 @@
 from abc import ABC, abstractmethod
 import beartype
+import numpy
+from scipy.spatial.distance import euclidean
 import pandas
 import geopandas
 from statistics import mean
 from .mapdata import MapData
+from scipy.interpolate import Rbf
+from .interpolators import DipDipDirectionInterpolator
+from .utils import create_points
+from .m2l_enums import Datatype
+from shapely.geometry import Point, shape
+from shapely import dwithin, shortest_line
```

Comment:
It might be best to do ```import shapely``` here, and then throughout the code use e.g., ```shapely.shortest_line```.
Having single function names without specifying the libraries may create clashes.
For example, shapely has an ```interpolate``` function, which could clash with the ```interpolate``` function in the interpolator abstract class (interpolator.py)

Reply:
Corrected as per your suggestion